misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/CodeCoverageUtils.java

```diff
@@ -170,9 +170,6 @@ private static boolean isRequiredFile(String path, String orgName, String module
             return false;
         } else if (path.contains("Frame") && path.contains(orgName)) {
             return false;
-        } else if (path.contains(
-                orgName + "/" + moduleName + "/" + version.replace(".", "_") + "/" + moduleName + ".class")) {
```

Comment:
Shall we remove the moduleName and version from the method signature if it is not used anywhere else?

Reply:
Fixed with 3c606e672ccc53455ac118feebe9102b76f4382c